lib/cdo/script_constants.rb

```diff
@@ -86,9 +88,6 @@ module ScriptConstants
       EXPLORE_DATA_1_2021_NAME = 'explore-data-1-2021'.freeze,
       SPELLING_BEE_2021_NAME = 'spelling-bee-2021'.freeze
     ],
-    aiml_2021: [
-      AIML_2021_NAME = 'aiml-2021'.freeze,
```

Comment:
Talking to @dju90, it sounds like AIML will be added to the translation pipeline sometime soon so maybe we should keep this constant?

Reply:
added!